resources/session02/homework/http_server.py

```diff
@@ -68,10 +130,13 @@ def server(log_buffer=sys.stderr):
                         content, mime_type = resolve_uri(uri)
                     except NameError:
                         response = response_not_found()
+                    except Exception as e:
+                        print('SERVER ERROR: {}'.format(e.message))
+                        response = server_error()
                     else:
                         response = response_ok(content, mime_type)
 
-                print('sending response', file=log_buffer)
+                print('sending response: {}'.format(response), file=log_buffer)
```

Comment:
This will result in **big** output to the terminal when responses include things like images.